lib/ansible/modules/network/meraki/meraki_switchport.py

```diff
@@ -33,21 +33,17 @@
     allowed_vlans:
         description:
         - List of VLAN numbers to be allowed on switchport.
```

Comment:
It is better to add to the documentation item list a line with the default value on creation time, for each parameter.
Otherwise people have to guess what will happen if it is unspecified.

Reply:
To clarify...you're saying I should call out the default, even if I'm not specifying a `default=` setting for the parameter?

Reply:
Yes, I think that would be helpful to users.
See: https://docs.ansible.com/ansible/devel/modules/aci_bd_module.html